src/layerelement.cpp

```diff
@@ -16,7 +16,6 @@
 #include "barline.h"
 #include "beam.h"
 #include "beatrpt.h"
-#include "btrem.h"
```

Comment:
Do you know why btrem was removed?

Reply:
I think, because `BTREM` isn't used in layerelement.cpp. 
I've put it back in.